plugins/inputs/suricata/suricata.go

```diff
@@ -36,18 +37,21 @@ type Suricata struct {
 
 // Description returns the plugin description.
 func (s *Suricata) Description() string {
-	return "Suricata stats plugin"
+	return "Suricata stats and alerts plugin"
 }
 
 const sampleConfig = `
-  ## Data sink for Suricata stats log
+  ## Data sink for Suricata stats and alerts logs
   # This is expected to be a filename of a
   # unix socket to be created for listening.
   source = "/var/run/suricata-stats.sock"
 
   # Delimiter for flattening field keys, e.g. subitem "alert" of "detect"
   # becomes "detect_alert" when delimiter is "_".
   delimiter = "_"
+  
+  # Detect alert logs 
+  alerts = false 
```

Comment:
You should leave default field values commented and use a double `#` for comments:
```suggestion
  
  ## Detect alert logs 
  # alerts = false 
```
I see it's also "wrong" in the lines above. So I'd be happy if you can correct those too. :-)